comm/src/test/scala/io/casperlabs/comm/gossiping/DownloadManagerSpec.scala

```
@@ -11,9 +22,23 @@ class DownloadManagerSpec extends WordSpecLike {
       "eventually download the full DAG" in (pending)
       "not exceed the limit on parallelism" in (pending)
     }
+
     "scheduled to download a block with missing dependencies" should {
-      "raise an error" in (pending)
+      val block  = arbitrary[Block].suchThat(_.getHeader.parentHashes.nonEmpty).sample.get
+      val source = arbitrary[Node].sample.get
+      val remote = MockGossipService.fromBlocks(Seq(block))
+
+      "raise an error" in TestFixture(remote = _ => remote) { manager =>
+        manager.scheduleDownload(summaryOf(block), source, false).attempt.map {
+          case Left(GossipError.MissingDependencies(_)) =>
```

I would prefer to have explicit assertion instead of just skipping it and failing for the unwanted case.

---

Okay I'll change it around.